applications/FluidDynamicsApplication/tests/AdjointQSVMSSensitivity2DTest/steady_cylinder_test_materials.json

```diff
@@ -0,0 +1,19 @@
+{
+    "properties": [
+        {
+            "model_part_name": "MainModelPart.Parts_Fluid",
+            "properties_id": 1,
+            "Material": {
+                "constitutive_law": {
+                    "name": "Newtonian2DLaw"
+                },
+                "Variables": {
+                    "DENSITY": 1.0E+00,
+                    "VISCOSITY": 1.0E-03,
+                    "DYNAMIC_VISCOSITY": 1.0E-03
```

Comment:
Why providing both? I'd provide only the one that is used (i.e. `DYNAMIC_VISCOSITY`)

Reply:
done :) [ copy pastet mistake ]